src/Symfony/Component/Config/Tests/Definition/EnumNodeTest.php

```diff
@@ -61,6 +62,47 @@ public function testFinalizeWithInvalidValue()
         $node->finalize('foobar');
     }
 
+    public function testFinalizeWithOnlyUnitEnumFqcnDoesntDoAnythingSpecial()
+    {
+        $node = new EnumNode('foo', null, [TestEnum::class]);
+
+        $this->expectException(InvalidConfigurationException::class);
+        $this->expectExceptionMessage('The value "foobar" is not allowed for path "foo". Permissible values: "Symfony\\\\Component\\\\Config\\\\Tests\\\\Fixtures\\\\TestEnum"');
```

Comment:
This error output is not user friendly. The enum class might even be an implementation detail that someone who writes a piece of configuration should not need to know about.

I would expect the backing values of the enum to be enumerated in the exception message.